README.md

```diff
@@ -159,7 +159,7 @@ const chromeless = new Chromeless({
 - [`exists(selector: string)`](docs/api.md#api-exists)
 - [`screenshot()`](docs/api.md#api-screenshot)
 - [`pdf(options?: PdfOptions)`](docs/api.md#api-pdf)
-- [`getHtml()`](docs/api.md#api-gethtml)
+- [`html()`](docs/api.md#api-html)
```

Comment:
Ha, I was actually going back and forth about the naming of `getHtml()` vs `html()` when I implemented this.

I originally called it `getHtml()` to avoid confusion with `setHtml()`, but I actually prefer `html()`, as it's more in line with the naming of `screenshot()` and `pdf()`.

I like this change 👍